### DISCUSSION TITLE
Bug on Any Large model - sometimes when i convert a video some scenes are in 3d and then a shift happens where it stops being in 3d  and becomes completely flat

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#108] [General]
hi, im experiencing an issue specifically with the any large model in iw3 where sometimes the video goes from 3d to becoming completly flat, i have a video showing the issue where

you can see an example of the issue in this video at the link here https://vimeo.com/930885724

n the beginning of the clip before the guy kills the security guard it was in 3d, after he walks in the hall if u pay attention the depth shifted and becomes less and less 3d to flat. Then when he leaves the building and he is talking to the guy on the outside its completely flat. 

this issue specifically happens on the any large model, i do NOT get this issue with any of the other models

the settings i used on this video was depth .92 your own size 3, convergence .5, .99 flicker reduction,  crf 9,  edge fix 1 negative foreground 2

**Comment by nagadomi** [comment]
Is it reproducible?
It means that if you convert the same video with the same settings, the same bug will occur around the same time each try.

A possible cause is that NaN is occurring on a specific frame image.

I have made a fix based on this guess, for now.
Let me know if the same problem occurs again after updating.
